infra/cifuzz/cifuzz.py

```diff
@@ -164,6 +167,27 @@ def run_fuzzers(project_name, fuzz_seconds, workspace):
     else:
       logging.info('Fuzzer %s, detected error: %s.', target.target_name,
                    stack_trace)
-      shutil.move(test_case, os.path.join(out_dir, 'testcase'))
+      shutil.move(test_case, os.path.join(bug_report_dir, 'test_case'))
+      parse_fuzzer_output(stack_trace, bug_report_dir)
       return True, True
   return True, False
+
+
+def parse_fuzzer_output(fuzzer_output, out_dir):
+  """Parses the fuzzer output from a fuzz target binary.
+
+  Args:
+    fuzzer_output: A fuzz target binary output string to be parsed.
+    out_dir: The location to store the parsed output files.
+  """
+  # Get index of key file points.
+  begin_summary = fuzzer_output.find('==ERROR')
+  end_summary = fuzzer_output.find('==ABORTING')
+  summary_str = fuzzer_output[begin_summary:end_summary]
```

Comment:
Shouldn't this be `fuzzer_output[begin_summary:end_summary+len(end)]` to include '==ABORTING'

Reply:
Fixed.